recipes/yaehmop/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "yaehmop" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/greglandrum/yaehmop/archive/master.tar.gz
```

Comment:
Please don't pull from master. A release on github should be tagged and then the package should pull the release tarball.

Reply:
I've poked the person upstream about making a github release we can target